.github/workflows/ci.yaml

```diff
@@ -0,0 +1,136 @@
+---
+
+name: 'CI'
```

Comment:
# ToDo

- [ ] Add to release job text an example inclusion details.

    tag, branch and git hash. actual details.